src/python/CRABInterface/Regexps.py

```diff
@@ -60,7 +60,7 @@
 RX_LFNPATH   = re.compile(r"^(?=.{0,500}$)%(subdir)s(/%(subdir)s)*/?$" % lfnParts)
 RX_HOURS   = re.compile(r"^\d{0,6}$") #should be able to erase the last 100 years with 6 digits
 RX_URL = re.compile(r"^(https?:\/\/)?([\da-z\.-]+)\.([a-z\.]{2,6})([\/\w :\.-])*$")
-RX_SCRIPTARGS = re.compile(r'^[+a-zA-Z0-9\-.,_:?/"]+=[a-zA-Z0-9\-=.,_:?/"()]+$')
+RX_SCRIPTARGS = re.compile(r'^[+a-zA-Z0-9\-.,_:?/"]+ ?= ?[a-zA-Z0-9\-=.,_:?/"()]+$')
```

Comment:
good. But can you write in English what the SCRIPTARGS syntax has to be for the documentation in 
https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
?
If we allow a list of the form `['a=1', 'c = 44', etc.]` with at most 1 blank around the `=` but not two... better make it clear.
I would see nothing wrong in going for the simple "no spaces allowed" and HC changes their configs if needed. But all those small uncertainties and small differences between one parameter and another cause a lot of pain to me when some user hits them and I have to dig and explain.